templates/skeleton/app/routes/account.orders._index.tsx

```diff
@@ -100,92 +101,19 @@ function EmptyOrders() {
   );
 }
 
-function OrderItem({order}: {order: OrderItemFragment}) {
+function OrderItem({order}: {order: any}) {
   return (
     <>
       <fieldset>
         <Link to={`/account/orders/${order.id}`}>
-          <strong>#{order.orderNumber}</strong>
+          <strong>#{order.number}</strong>
         </Link>
         <p>{new Date(order.processedAt).toDateString()}</p>
         <p>{order.financialStatus}</p>
-        <p>{order.fulfillmentStatus}</p>
```

Comment:
`fulfillment` is now an array and status is nested under each fulfillment. To keep the example simple, remove this functionality